# Author: Caspar van Leeuwen
# Company: SURF
# Python 3.14 is not an officially recognized classifier yet, even if the rich-argparse
# already claims to support it. As a result, hatchling refuses to build it
# Can be fixed by removing the classifier from the pyproject.toml
diff -Nru rich_argparse-1.7.1.orig/pyproject.toml rich_argparse-1.7.1/pyproject.toml
--- rich_argparse-1.7.1.orig/pyproject.toml	2025-06-16 16:56:11.057910803 +0200
+++ rich_argparse-1.7.1/pyproject.toml	2025-06-16 16:56:19.326904158 +0200
@@ -23,7 +23,6 @@
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: 3.13",
-  "Programming Language :: Python :: 3.14",
   "Topic :: Software Development :: User Interfaces",
 ]
 keywords = ["argparse", "rich", "help-formatter", "optparse"]
